corehq/util/metrics/tests/test_lockmeter.py

```
@@ -155,6 +160,19 @@ def test_del_untracked(self):
+    def test_local(self):
+        redis = get_redis_connection()
+        name = uuid1().hex
+        with Lock(redis, name, timeout=5) as redis_lock:
+            lock = MeteredLock(redis_lock, name)
+            self.assertEqual(type(lock.local), threading.local)
+
+    def test_no_local(self):
+        fake = FakeLock()
+        lock = MeteredLock(fake, "test")
+        with self.assertRaises(AttributeError):
+            lock.local
```

Ruff (0.8.2)

Found useless expression. Either assign it to a variable or remove it.

(B018)